02.explore/06.bank-and-shop/docs.md

```diff
@@ -39,13 +39,21 @@ You can locate all items that you purchase in the **Inventory** App.
 
 The Inventory app provides an interface to manage your transactions, purchases, and HFC. From the Inventory app, you can: 
 
-* View your recent purchases, sales, gifts and other transaction history
-* Change avatars, put on wearables and install/update apps
-* [Send HFC to your friends or anyone nearby](../socialize#send-hfc-to-others)
+* View your recent purchases, sales, gifts and other transaction history.
+* Change avatars, put on wearables and install/update apps.
+* [Send HFC to your friends or anyone nearby](../socialize#send-hfc-to-others).
 
 ![](inventory-app.png)
 
+>>>>> High Fidelity stores your Inventory key in the following locations. 
```

Comment:
So users who used to use the Inventory as the  Wallet had a key they used to access their Wallet from other machines. Since this was a confusing setup, we removed prompting users to save their keys and instead it is now saved on a user's local machine. If a user can't access their account anymore (it is a rare use case, like in the Manuscript Case), they can use this saved key. I've specified this in the doc note. Let me know if that works for you, or if you think it should be removed.